Formula/clblast.rb

```diff
@@ -0,0 +1,16 @@
+class Clblast < Formula
+  desc "Tuned OpenCL BLAS library"
+  homepage "https://github.com/CNugteren/CLBlast"
+  url "https://github.com/CNugteren/CLBlast/archive/1.3.0.tar.gz"
+  sha256 "fd5e06c261551366b46ccbc7841012cc69b58c74ed641db4f7a157c055d24223"
+  depends_on "cmake" => :build
```

Comment:
Blank line before and after `depends_on`

Reply:
Done